getsolus/packages
# teamspeak3: Update to 3.6.2

**Summary**

- Changelogs :
    - [3.6.2](https://community.teamspeak.com/t/teamspeak-3-client-3-6-2/43199)
    - [3.6.1](https://community.teamspeak.com/t/teamspeak-3-client-3-6-1/42048)
    - [3.6.0](https://community.teamspeak.com/t/teamspeak-3-client-3-6-0/41112)
- Resolves https://github.com/getsolus/packages/issues/1680
- Added hicolor icon
- Added appstream metainfo (Part of https://github.com/getsolus/packages/issues/1389)

**Test Plan**

- Lauch the app (Tested on all DEs)
- Connect to server and chat
- Verify metadata with `appstream-builder --packages-dir=. --include-failed -v`

**Checklist**


**Packager Note**

Still gives warning of missing library :
```
Missing library: libQt5WebSockets.so.5
Missing library: libwebp.so.6
```
`libQt5WebSockets.so.5` is provided by `qt5-websockets` (pkgconfig(Qt5Websockets))
`libwebp.so.6` is provided by **older** version of `libwebp` not available in repo
